.github/workflows/forest.yml

```diff
@@ -118,6 +118,14 @@ jobs:
       - name: Set permissions
         run: |
           chmod +x ~/.cargo/bin/forest*
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "^1.20"
+      - name: Install go-car
+        run: go install github.com/ipld/go-car/cmd/car@v0.0.0-20230704002448-4de031a1d8a8
```

Comment:
I like this versioning. Very agile.

Reply:
It's recommended to use `@latest` locally but I would rather use a fixed version on CI

Reply:
Yeah, I was just mocking the `0.0.0` version. You cant' have any expectations in such a case. :)